externals/ssl/src/CVMFS-CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+include ("${EXTERNALS_LIB_LOCATION}/CMake-Register_External_Lib.txt")
+
+set (SSL_ARCHIVE "${SSL_BUILTIN_LOCATION}/ssl_install/lib/libssl.a;${SSL_BUILTIN_LOCATION}/ssl_install/lib/libcrypto.a")
```

Comment:
Oh, I see.  Neat trick.

The dependency is still needed, I think, for parallel builds, to make sure that the libssl.a is available before it's time to link it.